app/src/main/kotlin/team/ommaya/wequiz/android/intro/join/JoinFragment.kt

```diff
@@ -0,0 +1,122 @@
+/*
+ * Designed and developed by "옴마야" Team 2023.
+ *
+ * Licensed under the MIT.
+ * Please see full license: https://github.com/mash-up-kr/WeQuiz-Android/blob/main/LICENSE
+ */
+
+package team.ommaya.wequiz.android.intro.join
+
+import android.os.Bundle
+import android.view.View
+import android.view.ViewGroup
+import androidx.core.widget.addTextChangedListener
+import androidx.navigation.fragment.findNavController
+import team.ommaya.wequiz.android.R
+import team.ommaya.wequiz.android.base.BaseViewBindingFragment
+import team.ommaya.wequiz.android.databinding.FragmentJoinBinding
+import team.ommaya.wequiz.android.utils.KeyboardVisibilityUtils
+import team.ommaya.wequiz.android.utils.isValidInputLengthRange
+
+class JoinFragment : BaseViewBindingFragment<FragmentJoinBinding>(FragmentJoinBinding::inflate) {
+    private lateinit var keyboardVisibilityUtils: KeyboardVisibilityUtils
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+
+        initView()
+        initKeyboardVisibilityUtils()
+    }
+
+    private fun initView() {
+        with(binding) {
+            etJoinInputNickname.addTextChangedListener {
+                setNextButtonEnable()
+            }
+
+            etJoinInputIntroduction.addTextChangedListener {
+                setNextButtonEnable()
+            }
+
+            etJoinInputNickname.setOnFocusChangeListener { _, hasFocus ->
+                if (hasFocus) {
+                    textInputLayoutJoinInputNickname.isCounterEnabled = true
+                    textInputLayoutJoinInputIntroduction.isCounterEnabled = false
+                } else {
+                    textInputLayoutJoinInputNickname.isCounterEnabled = false
+                }
```

Comment:
```suggestion
                textInputLayoutJoinInputNickname.isCounterEnabled = hasFocus
                if (hasFocus) {
                    textInputLayoutJoinInputIntroduction.isCounterEnabled = false
                }
```

이렇게 코드를 좀 더 줄일 수 있겠네용 참고만 해주세요

Reply:
얘는 사실 hasFocus 되었을때와 안되었을때 어떻게 동작하는지를 구분하려고 해놓은 코드긴한데 참고하겠습니다!

Reply:
수정 완료~